xbmc/utils/SystemInfo.cpp

```diff
@@ -72,6 +72,7 @@ using namespace winrt::Windows::System::Profile;
 #include <sys/param.h>
 #elif defined(TARGET_LINUX)
 #include <linux/version.h>
+#include <utils/SysfsUtils.h>
```

Comment:
change to `#include "utils/SysfsUtils.h"` and move it into the above block, as Kodi headers should be included before system headers (https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#6-headers)